requirements/extra-ml-requirements.txt

```diff
@@ -66,6 +66,8 @@ tiktoken
 tenacity
 # Required by mlflow.llama_index
 llama_index
+# Required for an agent example of mlflow.llama_index
+llama-index-agent-openai
```

Comment:
Can we add this into test requirements?

Reply:
I think `test-requirements.txt` does not flavor tests e.g. it doesn't contain `langchain`, `transformers`, etc.

Reply:
Can we add in ml-package-versions.yml

Reply:
No, we don't need this for cross-version test. It is only required for example test.